apps/concept_items/resource.py

```diff
@@ -30,9 +30,14 @@ class ConceptItemsResource(Resource):
             required=False,
             embeddable=False
         ),
-        'definition': {
+        'definition_plain': {
```

Comment:
I think it could be like `definition_text` so it's consistent with item, we have there `body_text` and `body_html`

Reply:
Makes sense. Thanks.